market/connector/adobe-acrobat-sign-connector/meta.json

```diff
@@ -1,21 +1,21 @@
 {
     "$schema": "https://json-schema.axonivy.com/market/10.0.0/meta.json",
-    "id": "adobe-esign-connector",
-    "name": "Adobe Esign Connector",
+    "id": "adobe-acrobat-connector",
+    "name": "Adobe Acrobat Connector",
     "description": "Use this connector to send, sign, track, and manage signature processes in Axon Ivy.",
     "type": "connector",
     "platformReview": "4.5",
-    "sourceUrl": "https://github.com/axonivy-market/adobe-esign-connector",
-    "statusBadgeUrl": "https://github.com/axonivy-market/adobe-esign-connector/actions/workflows/ci.yml/badge.svg",
+    "sourceUrl": "https://github.com/axonivy-market/adobe-acrobat-sign-connector",
+    "statusBadgeUrl": "https://github.com/axonivy-market/adobe-acrobat-sign-connector/actions/workflows/ci.yml/badge.svg",
     "language": "English",
     "industry": "Cross-Industry",
     "tags": [ "e-signature" ],
     "mavenArtifacts": [
         {
             "repoUrl": "https://maven.axonivy.com",
-            "name": "Adobe Esign Connector",
-            "groupId": "com.axonivy.connector.adobe.esign",
-            "artifactId": "adobe-esign-connector",
+            "name": "Adobe Acrobat Sign Connector",
+            "groupId": "com.axonivy.connector.adobe.acrobat.sign",
+            "artifactId": "adobe-acrobat-sign-connector",
```

Comment:
ahh here's the bug: @tutn-axonivy : the artifactId must reference the product -> therefore 'adobe-acrobat-sign-connector-product'

Reply:
see https://github.com/axonivy-market/adobe-acrobat-sign-connector/packages/2117882